paddle/fluid/inference/analysis/ir_pass_manager.cc

```diff
@@ -79,6 +79,10 @@ void IRPassManager::CreatePasses(Argument *argument,
     } else if (pass_name == "cpu_quantize_pass") {
       pass->Set("quant_var_scales",
                 new VarQuantScale(argument->quant_var_scales()));
+    } else if (pass_name == "cpu_bfloat16_placement_pass") {
```

Comment:
Why this change is needed?

Reply:
It should be added earlier because without it function in C-API `SetBfloat16Op` doesn't work.

Reply:
ok